README.md

```diff
@@ -136,7 +137,7 @@ Command     | Aliases      | Subcommands
 
 ## Semantic Versioning
 
-Byebug tries to follow [semantic versioning](https://semver.org) and tries to
+Byebug attempts to follow [semantic versioning](https://semver.org) as well as
```

Comment:
Bumping major version when backwards incompatible changes are released is part of semantic versioning, so I'm not sure as well is the best fit here? This section needs to be rewritten anyways, since this is not the exact policy I've been following, but I wanted to mention this in case you can think of a better wording.

Reply:
Thanks for the comment! I am a student as well as a Rails enthusiast and I love using Byebug.

How about:
"Byebug attempts to follow semantic versioning and bump major version only when backwards ..."?

Reply:
Yeah, I think that reads better! :+1: